draft-yoshino-wish.txt

```diff
@@ -238,6 +238,11 @@ Internet-DrafWiSH: A General Purpose Message Framing over B October 2016
    for flushing part of a large message payload without waiting for the
    total size of the message to be determined.
 
+   The CMP bit is used when a compression is in use.  Otherwise, it MUST
```

Comment:
I meant that this bit must always be zero when any compression is enabled for this WiSH body.

But, yeah, I think I understood your point. Here, we shouldn't talk about the concept of compression on/off for the whole WiSH body.

I'll rephase.